logging/system-test/sinks.test.js

```diff
@@ -0,0 +1,129 @@
+// Copyright 2015-2016, Google, Inc.
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+var gcloud = require('google-cloud');
+var uuid = require('node-uuid');
+var program = require('../sinks');
+
+var logging = gcloud.logging();
+var storage = gcloud.storage();
+
+var bucketName = 'nodejs-docs-samples-test-' + uuid.v4();
+var sinkName = 'nodejs-docs-samples-test-' + uuid.v4();
+
+describe('logging:sinks', function () {
+  before(function (done) {
+    storage.createBucket(bucketName, done);
+  });
+
+  after(function (done) {
+    logging.sink(sinkName).delete(function () {
+      // Don't check for error, the sink might already have been deleted
+      storage.bucket(bucketName).delete(function () {
+        // Don't check for error, the bucket might already have been deleted
+        done();
+      });
+    });
+  });
+
+  describe('createSink', function () {
+    it('should create a new sink', function (done) {
+      var options = {
+        name: sinkName,
+        destination: bucketName,
+        type: 'bucket'
+      };
+
+      program.createSink(options, function (err, sink) {
+        assert.ifError(err);
+        assert(sink, 'should have received the new sink');
+        assert.equal(sink.name, sinkName, 'should have received the new sink');
```

Comment:
Isn't this a more restrictive test than line 51?

My vote is for either a) removing line 51 entirely (because line 52 will still fail) or b) using a message along the lines of `sink is not null` for line 51's error.